### ISSUE TITLE
A11y_Azure Data Studio_Toolbar_FindNode_Zoom: At resize, the combo box is not visible completely

### ISSUE THREAD

**Issue by Truptik12** [microsoft/azuredatastudio#20782]
“[Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

### GitHubTags:
#A11yMAS; #A11yTCS; #A11ySev2; #SQL Azure Data Studio; #SH-SQL Azure Data Studio-Win32-OCT2022; #DesktopApp; #Zoom; #WCAG1.4.4; #FTP;

### Environment Details:
Application Name: Azure Data Studio
Windows Version: Windows 11

### Repro Steps:

1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Find Node" button and hit ENTER key.
8. Zoom the screen to 200% and observe the issue.

### Actual:
At resize, the combo box is not visible completely.

**This issue is also observed for the table in the "Properties" section in the below scenario**
**1)Toolbar - Properties**
1. Launch the "Azure data studio" application.
2. Connect with the provided "sqltools2019-3" SQL server.
3. TAB to "New Query" button and hit ENTER key.
4. Enter the SQL query and TAB to "Estimated Plan" button and hit ENTER key.
5. "Query Plan (Preview) screen appear.
6. TAB to the toolbar present at right side of the screen.
7. TAB to "Properties" button and hit ENTER key.
8. Zoom the screen to 200% and observe the issue.

### Expected Result:
At resize, the combo box should be visible completely.

### Attachment:
![image](https://user-images.githubusercontent.com/86710491/194820343-3122e7ad-4539-48d3-a5e0-0b0b73294239.png)

**Comment by aasimkhan30** [comment]
@Truptik12, can you please retest and close this issue?

**Comment by Truptik12** [comment]
Verified the bug in the below environment
Version: 1.40.0-insider (user setup)
Commit: https://github.com/microsoft/azuredatastudio/commit/d5cf088ba7b794859438fd2c1902249fd1e6a947
Date: 2022-10-26T05:45:47.459Z
VS Code: 1.67.0
Electron: 17.4.5
Chromium: 98.0.4758.141
Node.js: 16.13.0
V8: 9.8.177.13-electron.0
OS: Windows_NT x64 10.0.22621
The bug does not repro. Hence closing the bug. Please find the screenshot below.

![20782](https://user-images.githubusercontent.com/86710491/197973037-24baffb1-8f61-4a88-bf99-80c08f9305c8.png)
![20782 1](https://user-images.githubusercontent.com/86710491/197973053-01bc0eea-be28-4c0d-ab54-19fb659be420.png)
